cmd/cloud-controller-manager/app/apis/config/v1alpha1/doc.go

```diff
@@ -23,7 +23,7 @@ limitations under the License.
 // +k8s:conversion-gen=k8s.io/kubernetes/cmd/cloud-controller-manager/app/apis/config
 // +k8s:conversion-gen=k8s.io/component-base/config/v1alpha1
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/controller/apis/config/v1alpha1
-// +k8s:conversion-gen=k8s.io/kubernetes/pkg/controller/service/config/v1alpha1
+// +k8s:conversion-gen=k8s.io/cloud-provider/controllers/service/config/v1alpha1
```

Comment:
I'm confused... based on https://github.com/kubernetes/kubernetes/pull/90976#issuecomment-631633404 I thought we were not moving config struct locations

Reply:
The v1alpha1 types live in staging/src/k8s.io/kube-controller-manager/config/v1alpha1 (this is the case today as well). This just moves the generated code that was in pkg/controller/service/config/v1alpha1 to staging along with the controller. That seemed fine to me since none of the types are there (maybe this is still wrong though?) 

Before this comment https://github.com/kubernetes/kubernetes/pull/90976#issuecomment-631633404 I moved the config struct from `staging/src/k8s.io/kube-controller-manager` into `k8s.io/cloud-provider/controllers/service/config/v1alpha1`). I thought your comment was specifically for that so I reverted and left the versioned types in kube-controller-manager.

Reply:
> This just moves the generated code that was in pkg/controller/service/config/v1alpha1 to staging along with the controller.

The issue is that places v1alpha1 config-related code in a place that has no concept of a v1alpha1 config... that seems problematic.